src/components/CoverImage/CoverImage.jsx

```diff
@@ -0,0 +1,68 @@
+import "./coverimage.css";
+
+export const CoverImage = ({ singleCoverItem }) => {
+  const imageIndex = 0; //Index of the image to be displayed, choosing the first image in the array
+  let imgObjects = {
+    imgSrc: singleCoverItem.images[imageIndex].url, // Extract the URL of the album cover image
+    imgAlt: singleCoverItem.name, //// Set the alt attribute of the image to the album name
+  };
+
+  //Display the album cover image with alt text
+  //inserting a wrapper for the buttons and linking the svg files (the icons)
+  return (
+    <div className="cover-image">
+      <img src={imgObjects.imgSrc} alt={imgObjects.imgAlt} /> 
+
+      <div className="button-wrapper">
```

Comment:
The buttons could have been put into a separate component, because it the amount of code to build them is a little extensive.